Documentation/specs/runtime-configuration-file.md

```diff
@@ -25,8 +25,13 @@ The files are both JSON files stored in UTF-8 encoding. Below are sample files.
 ```json
 {
     "runtimeOptions": {
-        "gcServer": true,
-        "gcConcurrent": false,
+
+        "System.GC.Server": true,
+        "System.GC.Concurrent": false,
+        "System.Threading.ThreadPool.MinThreads": 4,
+        "System.Threading.ThreadPool.MaxThreads": 8,
+        "System.Threading.Thread.UseAllCpuGroups": true
+
         "framework": {
             "name": "Microsoft.DotNetCore",
```

Comment:
Would it be better to put these under `properties` subsection, so the host will simply take all keys and `ToString` the values and pass it to coreclr. This bypasses type check on the values specified and catching errors early but helps a older version of the host to still pass newer properties to coreclr as it doesn't get into the business of parsing known names and typed values but simply calls `ToString` on them.

Cc @jkotas